samples/interactive-tutorials/README.md

```diff
@@ -0,0 +1,84 @@
+# Google Retail Search
+
+## Java code samples
+
+The code here demonstartes how to consume Google Retail Search API in Java
+
+## Get started with Google Cloud Retail
+
+This step is required if this is the first Retail API Tutorial you run. Otherwise, you can skip it.
+
+### Select your project and enable the Retail API
+
+Google Cloud organizes resources into projects. This lets you collect all the related resources for
+a single application in one place.
+
+If you don't have a Google Cloud project yet or you're not the owner of an existing one, you can
+[create a new project](https://console.cloud.google.com/projectcreate).
+
+After the project is created, set your PROJECT_ID to a ```project``` variable.
+
+1. Run the following command in Terminal:
+    ```bash
+    gcloud config set project <YOUR_PROJECT_ID>
+    ```
+
+1. Check that the Retail API is enabled for your Project in
+   the [Admin Console](https://console.cloud.google.com/ai/retail/).
+
+### Set up authentication
+
+To run a code sample from the Cloud Shell, you need to authenticate. To do this, use the Application
+Default Credentials.
+
+1. Set your user credentials to authenticate your requests to the Retail API
+
+    ```bash
+    gcloud auth application-default login
+    ```
+
+1. Type `Y` and press **Enter**. Click the link in Terminal. A browser window should appear asking
+   you to log in using your Gmail account.
```

Comment:
Suggest changing this to "log in using your Google account"

Reply:
Done.